Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -148,8 +148,16 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  *  The user's shipping address. May be nil.
- */
-@property(nonatomic, readonly, nullable)STPAddress *shippingAddress;
+ *  If you've already collected a shipping address from your user, you may
+ *  prefill it by setting this property. When your user enters a new address,
+ *  PaymentContext will save it to the current customer object. When PaymentContext
+ *  loads, if you haven't manually set a prefilled value, any shipping information
+ *  saved on the customer will be used to prefill shippingAddress.
+ *  Note: by default, your user will still be prompted to verify a prefilled 
+ *  shipping address. To change this behavior, you can set 
+ *  `verifyPrefilledShippingAddress` to NO in your `STPPaymentConfiguration`.
+ */
+@property(nonatomic, nullable)STPAddress *shippingAddress;
```

Comment:
Ah, that's a good point – I can move `shippingAddress` to STPUserInformation. 

I'll try to unify/document our behavior for phone and email too.